service/controller.go

```diff
@@ -1964,7 +1967,7 @@ func (s *service) ControllerPublishVolume(
 	}
 	log.WithFields(fields).Info("Executing ControllerPublishVolume with following fields")
 	// flag createNFSExport()
-
+	isNVME := false
```

Comment:
Hi @utkarsh2296 If this doesn't take much of your time, can you please consider renaming this from isNVME to isNVMETcp ?
Just in case if we have to add the support of nvme fc protocol in future.
What do you say?